cmd/local-gadget/containers/containers.go

```diff
@@ -15,34 +15,145 @@
 package containers
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
 
+	"github.com/spf13/cobra"
+
+	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
 	"github.com/kinvolk/inspektor-gadget/cmd/local-gadget/utils"
+	containercollection "github.com/kinvolk/inspektor-gadget/pkg/container-collection"
 	localgadgetmanager "github.com/kinvolk/inspektor-gadget/pkg/local-gadget-manager"
-	"github.com/spf13/cobra"
 )
 
+type ContainerFlags struct {
+	noTrunc bool
+}
+
+type ContainerParser struct {
+	commonutils.BaseParser
+
+	containerFlags *ContainerFlags
+}
+
 func NewListContainersCmd() *cobra.Command {
-	var commonFlags utils.CommonFlags
+	var containerFlags ContainerFlags
+
+	availableColumns := map[string]int{
+		"runtime": 0,
+		"id":      0,
+		"name":    0,
+	}
```

Comment:
Shouldn't it include all columns from the container struct? 

Something like should work:

```
$ sudo ./local-gadget list-containers -o json
WARN[0000] Runtime enricher (cri-o): couldn't get current containers 
[
  {
    "ID": "11d5de97dbf1fabbcd93807fc172d8be422ab40bfc531fc6f106058d111eb527",
    "Pid": 5934,
    "OciConfig": null,
    "Bundle": "",
    "Mntns": 4026533360,
    "Netns": 4026533365,
    "CgroupPath": "/sys/fs/cgroup/unified/system.slice/docker-11d5de97dbf1fabbcd93807fc172d8be422ab40bfc531fc6f106058d111eb527.scope",
    "CgroupID": 2225,
    "CgroupV1": "/system.slice/docker-11d5de97dbf1fabbcd93807fc172d8be422ab40bfc531fc6f106058d111eb527.scope",
    "CgroupV2": "/system.slice/docker-11d5de97dbf1fabbcd93807fc172d8be422ab40bfc531fc6f106058d111eb527.scope",
    "Namespace": "default",
    "Podname": "local-registry",
    "Name": "local-registry",
    "Labels": null,
    "OwnerReference": null,
    "PodUID": "",
    "Runtime": "docker"
  }
]
$ sudo ./local-gadget list-containers -o custom-columns=id,netns
WARN[0000] Runtime enricher (cri-o): couldn't get current containers 
ID               <--- netns column not printed but it's available on the json.
11d5de97dbf1f    
```

Reply:
You are right. I think I would still omit the ones that we already know will always be nil. Do you agree? The documentation already says it will be always nil.

Reply:
Yes, I agree. I think we're missing the `omitempty` tag on the container structure declaration to avoid that behavior, but it's not something we need to make on this PR.